pkg/scyllaclient/client_agent.go

```diff
@@ -309,6 +309,19 @@ func (ni *NodeInfo) ScyllaObjectStorageEndpoint(provider backupspec.Provider) (s
 		"and in `scylla.yaml` object_storage_endpoints config", provider)
 }
 
+// SupportsSkipCleanupAndSkipReshape returns whether Scylla supports skip_cleanup and skip_reshape parameters
+// in /storage_service/sstables/{keyspace} endpoint as 2025.2.0 has a bug if this parameters are set to true -
+// https://github.com/scylladb/scylladb/issues/24913.
+func (ni *NodeInfo) SupportsSkipCleanupAndSkipReshape() (bool, error) {
+	// Detect master builds
+	if scyllaversion.MasterVersion(ni.ScyllaVersion) {
+		return true, nil
+	}
+
+	// Check ENT
+	return scyllaversion.CheckConstraint(ni.ScyllaVersion, "> 2025.2.0")
+}
```

Comment:
As mentioned [here](https://github.com/scylladb/scylla-manager/pull/4480#issuecomment-3072889171), we want to go into the direction of just using those params starting from Scylla version where they were released (2025.2.0). This is a Scylla side bug and we don't want/need to special case it here.